addons/sys_gestures/CfgMovesBasic.hpp

```diff
@@ -0,0 +1,20 @@
+class CfgMovesBasic {
+    class DefaultDie;
+    class ManActions {
+        acre_radio_helmet = "acre_radio_helmet";
+        acre_radio_helmet_noADS = "acre_radio_helmet_noADS";
+        acre_radio_vest = "acre_radio_vest";
+        acre_radio_vest_noADS = "acre_radio_vest_noADS";
+        acre_radio_stop = "acre_radio_stop";
+    };
+    class Actions {
+        class Default;
+        class NoActions: ManActions {
+            acre_radio_helmet[] = {"acre_radio_helmet","Gesture"};
+            acre_radio_helmet_noADS[] = {"acre_radio_helmet_noADS", "Gesture"};
+            acre_radio_vest[] = {"acre_radio_vest","Gesture"};
+            acre_radio_vest_noADS[] = {"acre_radio_vest_noADS", "Gesture"};
+            acre_radio_stop[] = {"acre_radio_stop","Gesture"};
```

Comment:
Those could use GVARS, not sure if needed 🤔

Reply:
I'm not happy with this but it's done 🍡